db/custom_seeds/notifications.rb

```diff
@@ -0,0 +1,269 @@
+# Notification seed file, to mock notifications for a specific user (ideally yourself).
+# To run the command, run:
+# rails db:seed:notifications USERNAME='your_username'
+# if no username is given, the script will use the last user
+
+NUM_TOTAL_STEPS = 11
+
+user = User.find_by(username: ENV["USERNAME"]) || User.last
+
+puts "1/#{NUM_TOTAL_STEPS} seeding follower notifications..."
+# three followers notification
+User.first(3).each do |follower|
+  follow = Follow.create!(followable: user, follower: follower)
+  Notification.send_new_follower_notification_without_delay(follow)
+end
+
+# two followers notification
+follow = User.fourth.follow(user)
+User.fifth.follow(user)
+Notification.create!(
+  notifiable: follow,
+  action: "Follow",
+  user: user,
+  json_data: {
+    user: Notifications.user_data(User.fourth),
+    aggregated_siblings: [
+      Notifications.user_data(User.fourth),
+      Notifications.user_data(User.fifth)
+    ]
+  }
+)
+
+# one follower notification
+follow = User.find(6).follow(user)
```

Comment:
I would avoid using specific IDs when possible like here. If there is no user with ID 6 the entire script is going to come to a halt. I think by removing specific user IDs the script will be more flexible and can be run on databases in any state, including if someone was testing user deletion and had deleted their ID 6 user. One suggestion for tackling this might be to get the list of ids at the beginning, `User.ids` and then sifting through that and using it to populate some of these.